README.md

```diff
@@ -848,74 +837,86 @@ A `Receipt` is an attestation of the [Result] and requested [Effect]s by a [Task
 
 Receipts MUST use the same version as the invocation that they contain.
 
-## 8.1 Schema
+## 8.1 Trace
```

Comment:
@expede minor: Calling this `Trace` may be confusing b/c we've also referred to `cause` as a trace itself? Again, super minor and probably not actually confusing, but just thinking aloud.

Reply:
@zeeshanlakhani haha yeah I have split feelings on this. I'll try to find a better name. Let me know if you think of something, too :)

Terms already taken:

* Output
* Receipt
* Result

Reply:
yeah, response and payload are not bad.